release notes/v1.2.0.md

```diff
@@ -12,23 +12,19 @@ k6 v1.2.0 is here 🎉! This release includes:
 
 _optional intro here_
 
-### Extensions support
+### Automatic extension resolution [#5019](https://github.com/grafana/k6/pull/5019), [#4811](https://github.com/grafana/k6/pull/4811), [#4812](https://github.com/grafana/k6/pull/4812), [#4921](https://github.com/grafana/k6/pull/4921), [#4922](https://github.com/grafana/k6/pull/4922)
 
-We have made multiple enhancements for running scripts with extensions:
+k6 extensions allow you to add custom functionality to your performance tests, such as connecting to databases, message queues, or specialized networking protocols. Previously, using extensions [required manual building](https://grafana.com/docs/k6/latest/extensions/build-k6-binary-using-go) of a custom k6 binary with the extensions compiled in. This new version introduces the _Automatic Extension Resolution_ functionality, which is enabled by default and automatically detects when your script imports extensions and handles the complexity of provisioning the right k6 binary for you.
 
-- [#5019](https://github.com/grafana/k6/pull/5019) Enables the feature by default. Also renames it as Automatic Extension Resolution and the environment variable to enables/disables it as `K6_AUTO_EXTENSION_RESOLUTION`.
-- [#4811](https://github.com/grafana/k6/pull/4811) Removes limitations for processing test scripts from stdin or using file extensions other than `.js` or `.tar`.
-- [#4812](https://github.com/grafana/k6/pull/4812) When running a test using a custom `k6` binary, the automatic extension resolution checks if the it includes all required extensions before attempting to provision a custom binary.
-' [#4921](https://github.com/grafana/k6/pull/4921) Removes the requirement for GCK6 authentication for using the automatic extension resolution, and enables the feature for local execution (i.e., the `k6 run` command).
-- [#4922](https://github.com/grafana/k6/pull/4922) Supports running tests using community extensions.
+```javascript
+TODO
```

Comment:
@pablochacin do you have any good suggest for this example? `xk6-faker`?

Reply:
We use something like this in the cloud docs:

```
import faker from "k6/x/faker";

export default function () {
  console.log(faker.person.firstName());
}
```